open-budget/app/scripts/entity_details.coffee

```diff
@@ -1,14 +1,17 @@
-define(["jquery", "backbone", "models", "templates"], ($, Backbone, models, JST) ->
+define(["jquery", "underscore", "backbone", "models", "templates"], ($, _, Backbone, models, JST) ->
     class EntityDetailsView extends Backbone.View
             events:
               'click .exemption-expander': 'toggleExemptionDetails'
 
             initialize: ->
-                @model.selectedEntity.on 'change:selected', =>
```

Comment:
@Bnaya @naamac @shayshei 

Did you make sure that the stand-alone entity page is not affected by this change?